### DISCUSSION TITLE
In what order are hosts taken for probing ?

### DISCUSSION THREAD

**Discussion by ipanda432** [projectdiscovery/httpx#271] [Q&A] [answered]
In what order does ```httpx -l doms.txt | tee -a liveurls.txt``` take hosts for probing. I have a domain which is on 10th line in the list of doms.txt , the above command has been running for almost 3hrs and i dont see the active host(verified manually) from line 10 in the output .

```Version: v1.0.5```

```Installation method : Binary```
doms.txt contains 88008347 unique hosts

**Comment by ehsandeep** [comment, marked as answer]
@fantasticpanthom

1. Make sure you update to latest version https://github.com/projectdiscovery/httpx/releases before debugging.
2. You can make use of `-debug` flag to see if something is wrong
3. httpx makes a parallel connections based on a defined number of theads.
